Framework/Algorithms/inc/MantidAlgorithms/CorelliPowderCalibrationLoad.h

```diff
@@ -0,0 +1,68 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2014 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#pragma once
```

Comment:
I always use this macro to avoid circular include, so there is a high probability that compiling could occur if this is not included (although I have never tested for Mantid myself).